README.md

```diff
@@ -83,6 +83,22 @@ In the index.html, include the core branding stylesheet:
 #### **avatar**
 `string` Avatar image URL. Default is the special provided Antwerp icon.
 
+#### **aria**
+`ChatbotMessageAriaLabels` Accessibility strings used throughout the chatbot UI:

```

Comment:
nadeel aan deze aanpak is dat er niet in fallback values voorzien wordt. Als de gebruiker dus niet waardes voor alle strings voorziet dan zijn er geen aria labels. (Kan ook een issue zijn als we bvb in een patch update een nieuwe aria label toevoegen, want die is dan blanco.)

ik denk dat er toch op een of andere manier in fallback values voorzien moet worden